data/filters/youtube-clarification.yaml

```diff
@@ -0,0 +1,13 @@
+title: Hide Youtube channel clarification
```

Comment:
I have other small tweaks I'd love to add but that don't really justify their own filter. Let's copy what we did for [google-search-cleanup](https://github.com/xvello/letsblockit/blob/main/data/filters/google-search-cleanup.yaml) and start a `youtube-cleanup` template that we can extend later on.

You can then add the rules under a dedicated param (although it's the only one for now):

```yaml
  - name: channel-clarification
    description: Hide the channel description box under videos and shorts.
    type: checkbox
    default: false
```

I'd prefer to keep it off as default, as this box can be used to alert on scams and hoaxes.

Reply:
As soon as I have more time I will make a new .yaml file as you suggested. 

Speaking of other small tweaks, I have a tweak to hide the text after the dislike, save, download, report, share and clip button. All those texts saying "share", "report", "dislike this video" or "download" could be missed and saves space. Note: the buttons are still visible. 
The rule is `www.youtube.com###info > * [id^="top-level-buttons"] [class*="button-renderer"] [id="text"]:not(:has-text(/\d/))` 

The last part `:not(:has-text(/\d/))` makes sure you can still see how many likes a video has.

Reply:
> Good idea! I had the same filter (without the `:not` part), but the recent redesign broke it and I didn't take the time to rework it. Again, I think the syntax could be simpler with a CSS selector, I'll check it out.

Yesterday YouTube made the new redesign public, but I still have the normal one. As soon as I have the new lay-out I will at a new rule that will fix this in this new lay-out.